v2/pages/adminAssessment/adminAssessment.sql

```diff
@@ -23,6 +23,22 @@ WHERE
     AND q.deleted_at IS NULL
 ORDER BY z.number, z.id, aq.number;
 
+-- BLOCK question_stats
```

Comment:
See the [developer's guide section](https://github.com/PrairieLearn/PrairieLearn/blob/master/v2/doc/dev-guide.md#sql-usage) on SQL formatting and conventions, as well as the [DB schema section](https://github.com/PrairieLearn/PrairieLearn/blob/master/v2/doc/dev-guide.md#db-schema) for table name abbreviations.